bin/site-specific-install.sh

```diff
@@ -26,27 +26,39 @@ misuse() {
 [ -z "$DISTRIBUTION" ] && misuse DISTRIBUTION
 [ -z "$VERSION" ] && misuse VERSION
 [ -z "$DEVELOPMENT_INSTALL" ] && misuse DEVELOPMENT_INSTALL
+[ -z "$DOCKER" ] && misuse DOCKER
+[ -z "$INSTALL_DB" ] && misue INSTALL_DB
+[ -z "$INSTALL_POSTFIX" ] && misuse INSTALL_POSTFIX
 
 add_locale cy_GB
 add_locale nb_NO
 add_locale de_CH
 
-install_postfix
+if [ $INSTALL_POSTFIX = true ]; then
+    install_postfix
+fi
 
-if [ ! "$DEVELOPMENT_INSTALL" = true ]; then
+if [ ! "$DEVELOPMENT_INSTALL" = true ] && [ ! "$DOCKER" = true ]; then
     install_nginx
     add_website_to_nginx
     # Check out the current released version
     su -l -c "cd '$REPOSITORY' && git checkout '$VERSION' && git submodule update" "$UNIX_USER"
```

Comment:
I realise the checked out branch can now be set with BRANCH_OVERRIDE in the parent script and then again here with VERSION_OVERRIDE, but it seems a bit odd for the docker install not to run this line (which feels like a line you'd only want to bypass on a DEVELOPMENT_INSTALL).

Reply:
Yes - good point, we want the public image to be the latest tagged release.

Reply:
Hmm, just thinking this might not work without an override at the moment as-is until at least `docker.preinit` is in the default `$VERSION` and that gets copied over after the scripts are run.

Reply:
It's only an issue before the initial merge, after that we'd want a copy consistent with the tagged release I think unless we were building test images in which case we can add VERSION_OVERRIDE while building as you suggest.

I'll build the first version of the container user `git show`, but omit this from the final version.

Reply:
Hmm, or perhaps include the `git show` until we have a tagged release that contains these commits.

Reply:
Yeah, don't really think it matters - could fetch from github like the install-site even ;)